code/modules/projectiles/gun.dm

```diff
@@ -21,7 +21,14 @@
 	item_flags = NEEDS_PERMIT
 	attack_verb_continuous = list("strikes", "hits", "bashes")
 	attack_verb_simple = list("strike", "hit", "bash")
-
+	light_range = 3
+	light_system = OVERLAY_LIGHT
```

Comment:
If you make this
```suggestion
	light_system = OVERLAY_LIGHT_DIRECTIONAL
```
Then the gun muzzle will be cone shaped in the direction of the gun shot, which might look nicer.